src/style.css

```
@@ -1 +1,65 @@
-/* add styles here */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1::after {
+  content: "";
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  position: absolute;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--2 li:nth-child(-1n + 2)::after {
```

```suggestion
.stars--2 li:nth-child(-1n + 2)::after,
... {
```
Consider using commas to group your selectors and avoid repeating code